resolwe/permissions/tests/test_data.py

```diff
@@ -62,9 +62,9 @@ def test_post(self):
         d = Data.objects.get(pk=resp.data['id'])
         self.assertTrue(datetime.now() - d.modified < timedelta(seconds=1))
         self.assertTrue(datetime.now() - d.created < timedelta(seconds=1))
-        self.assertEqual(d.status, 'RE')
-        self.assertEqual(d.started, None)
-        self.assertEqual(d.finished, None)
+        self.assertEqual(d.status, 'OK')
+        self.assertTrue(datetime.now() - d.started < timedelta(seconds=1))
```

Comment:
Sorry, not relevant here, because this is a test with specific time zone setting. Will delete.